[2025-09-02T21:16:47.169+0800] INFO - local_task_job_runner.py:123 - _execute() - ::group::Pre task execution logs
[2025-09-02T21:16:47.275+0800] INFO - taskinstance.py:2613 - are_dependencies_met() - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_parallel_dag.end scheduled__2025-09-02T12:00:00+00:00 [queued]>
[2025-09-02T21:16:47.310+0800] INFO - taskinstance.py:2613 - are_dependencies_met() - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_parallel_dag.end scheduled__2025-09-02T12:00:00+00:00 [queued]>
[2025-09-02T21:16:47.311+0800] INFO - taskinstance.py:2866 - _check_and_change_state_before_execution() - Starting attempt 1 of 2
[2025-09-02T21:16:47.357+0800] INFO - taskinstance.py:2889 - _check_and_change_state_before_execution() - Executing <Task(BashOperator): end> on 2025-09-02 12:00:00+00:00
[2025-09-02T21:16:47.373+0800] INFO - standard_task_runner.py:72 - _start_by_fork() - Started process 445 to run task
[2025-09-02T21:16:47.391+0800] INFO - standard_task_runner.py:104 - _start_by_fork() - Running: ['***', 'tasks', 'run', 'example_parallel_dag', 'end', 'scheduled__2025-09-02T12:00:00+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/example_parallel_dag.py', '--cfg-path', '/tmp/tmpiqrh6g9q']
[2025-09-02T21:16:47.394+0800] INFO - standard_task_runner.py:105 - _start_by_fork() - Job 31: Subtask end
[2025-09-02T21:16:47.657+0800] INFO - task_command.py:467 - task_run() - Running <TaskInstance: example_parallel_dag.end scheduled__2025-09-02T12:00:00+00:00 [running]> on host a246a11defc8
[2025-09-02T21:16:48.387+0800] INFO - taskinstance.py:3132 - _execute_task_with_callbacks() - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data-team' AIRFLOW_CTX_DAG_ID='example_parallel_dag' AIRFLOW_CTX_TASK_ID='end' AIRFLOW_CTX_EXECUTION_DATE='2025-09-02T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-02T12:00:00+00:00'
[2025-09-02T21:16:48.392+0800] INFO - taskinstance.py:731 - _execute_callable() - ::endgroup::
[2025-09-02T21:16:48.469+0800] INFO - subprocess.py:78 - run_command() - Tmp dir root location: /tmp
[2025-09-02T21:16:48.475+0800] INFO - subprocess.py:88 - run_command() - Running command: ['/usr/bin/bash', '-c', 'echo "所有平行任務執行完成！"']
[2025-09-02T21:16:48.520+0800] INFO - subprocess.py:99 - run_command() - Output:
[2025-09-02T21:16:48.620+0800] INFO - subprocess.py:106 - run_command() - 所有平行任務執行完成！
[2025-09-02T21:16:48.621+0800] INFO - subprocess.py:110 - run_command() - Command exited with return code 0
[2025-09-02T21:16:48.772+0800] INFO - taskinstance.py:340 - _run_raw_task() - ::group::Post task execution logs
[2025-09-02T21:16:48.773+0800] INFO - taskinstance.py:352 - _run_raw_task() - Marking task as SUCCESS. dag_id=example_parallel_dag, task_id=end, run_id=scheduled__2025-09-02T12:00:00+00:00, execution_date=20250902T120000, start_date=20250902T131647, end_date=20250902T131648
[2025-09-02T21:16:48.932+0800] INFO - local_task_job_runner.py:266 - handle_task_exit() - Task exited with return code 0
[2025-09-02T21:16:49.043+0800] INFO - taskinstance.py:3895 - _schedule_downstream_tasks() - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-02T21:16:49.048+0800] INFO - local_task_job_runner.py:245 - _execute() - ::endgroup::
